tests/integration/golang/aim/namespace/flows/app_test.go

```diff
@@ -0,0 +1,280 @@
+//go:build integration
+
+package flows
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/request"
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/common"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type AppFlowTestSuite struct {
+	helpers.BaseTestSuite
+}
+
+func TestAppFlowTestSuite(t *testing.T) {
+	suite.Run(t, &AppFlowTestSuite{})
+}
+
+func (s *AppFlowTestSuite) TearDownTest() {
+	assert.Nil(s.T(), s.NamespaceFixtures.UnloadFixtures())
+}
+
+func (s *AppFlowTestSuite) Test_Ok() {
+	tests := []struct {
+		name           string
+		namespace1Code string
+		namespace2Code string
+	}{
+		{
+			name:           "TestCustomNamespaces",
+			namespace1Code: "namespace-1",
+			namespace2Code: "namespace-2",
+		},
+		{
+			name:           "TestExplicitDefaultAndCustomNamespaces",
+			namespace1Code: "default",
+			namespace2Code: "namespace-1",
+		},
+		{
+			name:           "TestImplicitDefaultAndCustomNamespaces",
+			namespace1Code: "",
+			namespace2Code: "namespace-1",
+		},
+	}
+
+	for _, tt := range tests {
+		s.T().Run(tt.name, func(T *testing.T) {
+			defer func() {
+				assert.Nil(s.T(), s.NamespaceFixtures.UnloadFixtures())
+			}()
+
+			// setup namespaces
+			for _, nsCode := range []string{"default", tt.namespace1Code, tt.namespace2Code} {
+				// ignore errors here since default exists on first run
+				//nolint:errcheck
+				_, _ = s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
```

Comment:
can we please in any case check all errors?

Reply:
yep and it will be really nice to clean and then start.

Reply:
done